content/learn/quick-start/getting-started/ecs.md

```diff
@@ -116,7 +116,7 @@ Now register the startup system like this:
 fn main() {
     App::new()
         .add_systems(Startup, add_people)
-        .add_systems(Update, hello_world)
+        .add_systems(Update, greet_people)
```

Comment:
At this point in the code, `greet_people` hasn't been introduced, and `hello_world` is removed explicitly in the resources chapter of quick start. I'm not sure this change is warranted per se.

Reply:
Yeah i think we should leave it